etc/run-auth-ldap.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# see run-jupyterhub.sh for setting up local mail delivery
+export SIREPO_FROM_EMAIL='support@radiasoft.net'
+export SIREPO_FROM_NAME='RadiaSoft Support'
+export SIREPO_SMTP_PASSWORD='vagrant'
+# POSIT: same as sirepo.smtp.DEV_SMTP_SERVER
+export SIREPO_SMTP_SERVER='dev'
+export SIREPO_SMTP_USER='vagrant'
+export SIREPO_AUTH_METHODS='email:guest:ldap'
+export PYKERN_PKDEBUG_WANT_PID_TIME=1
+
+# primary cause of failed wheel generation
+pip install --upgrade pip
+echo "upgraded pip"
+
+# install prerequisites
+sudo yum groupinstall "Development tools"
+sudo yum install openldap-devel python-devel
+echo "installed prerequisites"
```

Comment:
These types of echos are not useful. In general, things should be silent unless they fail.